src/styles/_catalog.scss

```
@@ -1,22 +1,33 @@
+$baseCardWidth: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-gap: 48px;
+  justify-content: center;
+
+  @media (max-width: 488px) {
+    grid-template-columns: $baseCardWidth;
+  }
```

```suggestion
  display: grid;
  grid-gap: 48px;
  justify-content: center;
  grid-template-columns: $baseCardWidth;
```

Not necessary to create another media query, just set this value as the default 🙌